config/browserstack/wdio.android.bs.app.conf.js

```diff
@@ -0,0 +1,80 @@
+const { config } = require('../wdio.shared.conf');
+const browserstack = require('browserstack-local');
```

Comment:
As far as I know this is not needed, the local service is only needed if you want to test websites that are on your local machine / you internal network. Because this app doesn't have any connection with that I think it should be removed